File: src/app/model/physician-availability.ts
export class PhysicianAvailability{
    constructor(
        public id:number,
        public email:string,
        public date:string[],
        public availability:boolean,

    ){
        this.id=id ||0;
        this.email=email||"";
        this.date=date || [];
        this.availability=availability||false;
    }
} 

File: physician-availability-service/src/main/java/com/revature/physicianavailabilityservice/model/PhysicianAvailability.java
package com.revature.physicianavailabilityservice.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "physician_availability")
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PhysicianAvailability {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;
	private String email;
	private String[] date;
	private boolean availability;

}
